app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -14,6 +15,12 @@ module HyraxHelperBehavior
     include Hyrax::ChartsHelper
     include Hyrax::DashboardHelperBehavior
 
+    # Which translations are available for the user to select
+    # @return [Hash<String,String>] locale abbreviations as keys and flags as values
+    def available_translations
+      { 'en' => '🇬🇧', 'es' => '🇪🇸' }
```

Comment:
This instead? https://github.com/projectblacklight/demo.projectblacklight.org/pull/2